ast/parser.go

```diff
@@ -352,34 +352,56 @@ func (p *Parser) Parse() ([]Statement, []*Comment, Errors) {
 
 func (p *Parser) parseAnnotations(stmts []Statement) []Statement {
 
+	annotStmts, errs := parseAnnotations(p.s.comments)
+	if len(errs) > 0 {
+		for _, err := range errs {
+			p.error(err.Location, err.Message)
+		}
+	}
```

Comment:
[nit] 
```suggestion
	for _, err := range errs {
		p.error(err.Location, err.Message)
	}
```

Buit it's OK, with the current code you'll at least not have to think twice about `len(errs)`.